<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">


  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">



<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/animate.css@3.1.1/animate.min.css" integrity="sha256-PR7ttpcvz8qrF57fur/yAx1qXMFJeJFiA6pSzWi0OIE=" crossorigin="anonymous">

<script class="next-config" data-name="main" type="application/json">{"hostname":"mmyoung.github.io","root":"/","images":"/images","scheme":"Gemini","darkmode":false,"version":"8.8.1","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12},"copycode":false,"bookmark":{"enable":false,"color":"#222","save":"auto"},"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"stickytabs":false,"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"fadeInDown","post_body":"fadeInDown","coll_header":"fadeInLeft","sidebar":"fadeInUp"}},"prism":false,"i18n":{"placeholder":"Searching...","empty":"We didn't find any results for the search: ${query}","hits_time":"${hits} results found in ${time} ms","hits":"${hits} results found"}}</script><script src="/js/config.js"></script>
<meta name="description" content="This blog is a simple summary for the published proteomic sequencing data in Alzheimer’s disease. The sample number, brain region used for sequencing and specific methods are recorded for further refe">
<meta property="og:type" content="article">
<meta property="og:title" content="Proteomic sequencing in Alzheimer&#39;s disease">
<meta property="og:url" content="https://mmyoung.github.io/2021/11/04/Proteomic-sequencing-in-Alzheimer-s-disease/index.html">
<meta property="og:site_name" content="Lin&#39;s blog">
<meta property="og:description" content="This blog is a simple summary for the published proteomic sequencing data in Alzheimer’s disease. The sample number, brain region used for sequencing and specific methods are recorded for further refe">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://mmyoung.github.io/2021/11/03/Proteomic-sequencing-in-Alzheimer-s-disease/proteome-seq%20in%20AD.png">
<meta property="article:published_time" content="2021-11-03T19:53:49.000Z">
<meta property="article:modified_time" content="2022-01-09T15:10:23.658Z">
<meta property="article:author" content="Lin">
<meta property="article:tag" content="Alzheimer&#39;s disease">
<meta property="article:tag" content="proteome sequencing">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://mmyoung.github.io/2021/11/03/Proteomic-sequencing-in-Alzheimer-s-disease/proteome-seq%20in%20AD.png">


<link rel="canonical" href="https://mmyoung.github.io/2021/11/04/Proteomic-sequencing-in-Alzheimer-s-disease/">



<script class="next-config" data-name="page" type="application/json">{"sidebar":"","isHome":false,"isPost":true,"lang":"en","comments":true,"permalink":"https://mmyoung.github.io/2021/11/04/Proteomic-sequencing-in-Alzheimer-s-disease/","path":"2021/11/04/Proteomic-sequencing-in-Alzheimer-s-disease/","title":"Proteomic sequencing in Alzheimer's disease"}</script>

<script class="next-config" data-name="calendar" type="application/json">""</script>
<title>Proteomic sequencing in Alzheimer's disease | Lin's blog</title>
  




  <noscript>
    <link rel="stylesheet" href="/css/noscript.css">
  </noscript>
</head>

<body itemscope itemtype="http://schema.org/WebPage" class="use-motion">
  <div class="headband"></div>

  <main class="main">
    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar" role="button">
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <i class="logo-line"></i>
      <h1 class="site-title">Lin's blog</h1>
      <i class="logo-line"></i>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>



<nav class="site-nav">
  <ul class="main-menu menu">
        <li class="menu-item menu-item-home"><a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a></li>
        <li class="menu-item menu-item-about"><a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>About</a></li>
        <li class="menu-item menu-item-tags"><a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>Tags</a></li>
        <li class="menu-item menu-item-categories"><a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>Categories</a></li>
  </ul>
</nav>




</div>
        
  
  <div class="toggle sidebar-toggle" role="button">
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
  </div>

  <aside class="sidebar">

    <div class="sidebar-inner sidebar-nav-active sidebar-toc-active">
      <ul class="sidebar-nav">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <div class="sidebar-panel-container">
        <!--noindex-->
        <div class="post-toc-wrap sidebar-panel">
            <div class="post-toc animated"><ol class="nav"><li class="nav-item nav-level-4"><a class="nav-link" href="#1-Quantification-of-the-Brain-Proteome-in-Alzheimer%E2%80%99s-Disease-Using-Multiplexed-Mass-Spectrometry-2014-Journal-of-Proteome-Research"><span class="nav-number">1.</span> <span class="nav-text">1. Quantification of the Brain Proteome in Alzheimer’s Disease Using Multiplexed Mass Spectrometry. (2014. Journal of Proteome Research.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#2-Deep-proteomic-network-analysis-of-Alzheimer%E2%80%99s-disease-brain-reveals-alterations-in-RNA-binding-proteins-and-RNA-splicing-associated-with-disease-2018-Molecular-Neurodegeneration"><span class="nav-number">2.</span> <span class="nav-text">2.  Deep proteomic network analysis of Alzheimer’s disease brain reveals alterations in RNA binding proteins and RNA splicing associated with disease. (2018. Molecular Neurodegeneration.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#3-Quantitative-proteomics-reveals-distinct-composition-of-amyloid-plaques-in-Alzheimer%E2%80%99s-disease-2018-Alzheimer%E2%80%99s-disease-amp-Dementia"><span class="nav-number">3.</span> <span class="nav-text">3. Quantitative proteomics reveals distinct composition of amyloid plaques in Alzheimer’s disease. (2018. Alzheimer’s disease &amp; Dementia.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#4-Proteomic-Atlas-of-the-Human-Brain-in-Alzheimer%E2%80%99s-Disease-2019-Jounral-of-Proteome-Research"><span class="nav-number">4.</span> <span class="nav-text">4. Proteomic Atlas of the Human Brain in Alzheimer’s Disease. (2019. Jounral of Proteome Research.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#5-Molecular-Network-Analysis-of-the-Urinary-Proteome-of-Alzheimer%E2%80%99s-Disease-Patients-2019-Dementia-amp-Geriatric-Cognitive-Disorders"><span class="nav-number">5.</span> <span class="nav-text">5. Molecular Network Analysis of the Urinary Proteome of Alzheimer’s Disease Patients. (2019. Dementia &amp; Geriatric Cognitive Disorders.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#6-Comparative-profiling-of-the-synaptic-proteome-from-Alzheimer%E2%80%99s-disease-patients-with-focus-on-the-APOE-genotype-2019-Acta-Neuropathologica-Communications"><span class="nav-number">6.</span> <span class="nav-text">6. Comparative profiling of the synaptic proteome from Alzheimer’s disease patients with focus on the APOE genotype. (2019.Acta Neuropathologica Communications.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#7-Rapamycin-regulates-cholesterol-biosynthesis-and-cytoplasmic-ribosomal-proteins-in-hippocampus-and-temporal-lobe-of-APP-PS1-mouse-2019-Journal-of-Neurological-Sciences"><span class="nav-number">7.</span> <span class="nav-text">7. Rapamycin regulates cholesterol biosynthesis and cytoplasmic ribosomal proteins in hippocampus and temporal lobe of APP&#x2F;PS1 mouse. (2019. Journal of Neurological Sciences.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#8-Large-scale-proteomic-analysis-of-Alzheimer%E2%80%99s-disease-brain-and-cerebrospinal-fluid-reveals-early-changes-in-energy-metabolism-associated-with-microglia-and-astrocyte-activation-2020-Nature-Medicine"><span class="nav-number">8.</span> <span class="nav-text">8. Large-scale proteomic analysis of Alzheimer’s disease brain and cerebrospinal fluid reveals early changes in energy metabolism associated with microglia and astrocyte activation. (2020. Nature Medicine.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#9-Integrated-proteomics-reveals-brain-based-cerebrospinal-fluid-biomarkers-in-asymptomatic-and-symptomatic-Alzheimer%E2%80%99s-disease-2020-Science-Advance"><span class="nav-number">9.</span> <span class="nav-text">9. Integrated proteomics reveals brain-based cerebrospinal fluid biomarkers in asymptomatic and symptomatic Alzheimer’s disease. (2020. Science Advance.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#10-Deep-Multilayer-Brain-Proteomics-Identifies-Molecular-Networks-in-Alzheimer%E2%80%99s-Disease-Progression-2020-Neuron"><span class="nav-number">10.</span> <span class="nav-text">10. Deep Multilayer Brain Proteomics Identifies Molecular Networks in Alzheimer’s Disease Progression. (2020. Neuron.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#11-Proteomic-and-Transcriptomic-Analyses-Reveal-Pathological-Changes-in-the-Entorhinal-Cortex-Region-that-Correlate-Well-with-Dysregulation-of-Ion-Transport-in-Patients-with-Alzheimer%E2%80%99s-Disease-2021-Molecular-Neurobiology"><span class="nav-number">11.</span> <span class="nav-text">11. Proteomic and Transcriptomic Analyses Reveal Pathological Changes in the Entorhinal Cortex Region that Correlate Well with Dysregulation of Ion Transport in Patients with Alzheimer’s Disease. (2021. Molecular Neurobiology.)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#12-Quantitative-proteomic-analysis-of-the-frontal-cortex-in-Alzheimer%E2%80%99s-disease-2020-Neurochemistry"><span class="nav-number">12.</span> <span class="nav-text">12. Quantitative proteomic analysis of the frontal cortex in Alzheimer’s disease. (2020. Neurochemistry.)</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#References"><span class="nav-number"></span> <span class="nav-text">References</span></a></div>
        </div>
        <!--/noindex-->

        <div class="site-overview-wrap sidebar-panel">
          <div class="site-author site-overview-item animated" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Lin</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap site-overview-item animated">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
        <a href="/archives/">
          <span class="site-state-item-count">4</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
          <a href="/categories/">
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
          <a href="/tags/">
        <span class="site-state-item-count">7</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>



        </div>
      </div>
        <div class="back-to-top animated" role="button" aria-label="Back to top">
          <i class="fa fa-arrow-up"></i>
          <span>0%</span>
        </div>
    </div>
  </aside>
  <div class="sidebar-dimmer"></div>


    </header>

    

<noscript>
  <div class="noscript-warning">Theme NexT works best with JavaScript enabled</div>
</noscript>


    <div class="main-inner post posts-expand">


  


<div class="post-block">
  
  

  <article itemscope itemtype="http://schema.org/Article" class="post-content" lang="en">
    <link itemprop="mainEntityOfPage" href="https://mmyoung.github.io/2021/11/04/Proteomic-sequencing-in-Alzheimer-s-disease/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="Lin">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Lin's blog">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          Proteomic sequencing in Alzheimer's disease
        </h1>

        <div class="post-meta-container">
          <div class="post-meta">
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-calendar"></i>
      </span>
      <span class="post-meta-item-text">Posted on</span>

      <time title="Created: 2021-11-04 03:53:49" itemprop="dateCreated datePublished" datetime="2021-11-04T03:53:49+08:00">2021-11-04</time>
    </span>
      <span class="post-meta-item">
        <span class="post-meta-item-icon">
          <i class="far fa-calendar-check"></i>
        </span>
        <span class="post-meta-item-text">Edited on</span>
        <time title="Modified: 2022-01-09 23:10:23" itemprop="dateModified" datetime="2022-01-09T23:10:23+08:00">2022-01-09</time>
      </span>
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-folder"></i>
      </span>
      <span class="post-meta-item-text">In</span>
        <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
          <a href="/categories/research-summary/" itemprop="url" rel="index"><span itemprop="name">research summary</span></a>
        </span>
          , 
        <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
          <a href="/categories/research-summary/about-disease/" itemprop="url" rel="index"><span itemprop="name">about disease</span></a>
        </span>
    </span>

  
</div>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">
        <p>This blog is a simple summary for the published proteomic sequencing data in Alzheimer’s disease. The sample number, brain region used for sequencing and specific methods are recorded for further reference.<br>This summary is based on the researches published untill the blog release date.</p>
<span id="more"></span>

<p>Long story short, the samples used for proteome sequencing in AD patients are: </p>
<img src="/2021/11/03/Proteomic-sequencing-in-Alzheimer-s-disease/proteome-seq in AD.png" alt="tissues that are used for proteome sequencing" width="400" height="400">

<hr>
<p>The detailed materials and methods used in each research are as following:</p>
<h4 id="1-Quantification-of-the-Brain-Proteome-in-Alzheimer’s-Disease-Using-Multiplexed-Mass-Spectrometry-2014-Journal-of-Proteome-Research"><a href="#1-Quantification-of-the-Brain-Proteome-in-Alzheimer’s-Disease-Using-Multiplexed-Mass-Spectrometry-2014-Journal-of-Proteome-Research" class="headerlink" title="1. Quantification of the Brain Proteome in Alzheimer’s Disease Using Multiplexed Mass Spectrometry. (2014. Journal of Proteome Research.)"></a>1. <a target="_blank" rel="noopener" href="https://pubs.acs.org/doi/10.1021/pr401202d">Quantification of the Brain Proteome in Alzheimer’s Disease Using Multiplexed Mass Spectrometry</a>. (2014. Journal of Proteome Research.)</h4><ul>
<li><strong>Subjects:</strong> <span style="font-size:0.8em;"><mark>Temporal neocortex</mark> of Alzheimer’s disease (AD, n=10) patients and non-AD (n=5) Individuals.</span></li>
<li><strong>Methods:</strong> <span style="font-size:0.8em;">shotgun mass spectrometry based on a stable isotope dimethyl labeling.</span></li>
<li><strong>Results:</strong> A total of 827 unique proteins were identified and quantitated. A total of 69 proteins showed different levels (p-value &lt; 0.05) in AD versus control brain samples.</li>
</ul>
<h4 id="2-Deep-proteomic-network-analysis-of-Alzheimer’s-disease-brain-reveals-alterations-in-RNA-binding-proteins-and-RNA-splicing-associated-with-disease-2018-Molecular-Neurodegeneration"><a href="#2-Deep-proteomic-network-analysis-of-Alzheimer’s-disease-brain-reveals-alterations-in-RNA-binding-proteins-and-RNA-splicing-associated-with-disease-2018-Molecular-Neurodegeneration" class="headerlink" title="2.  Deep proteomic network analysis of Alzheimer’s disease brain reveals alterations in RNA binding proteins and RNA splicing associated with disease. (2018. Molecular Neurodegeneration.)"></a>2.  <a target="_blank" rel="noopener" href="https://molecularneurodegeneration.biomedcentral.com/articles/10.1186/s13024-018-0282-4">Deep proteomic network analysis of Alzheimer’s disease brain reveals alterations in RNA binding proteins and RNA splicing associated with disease</a>. (2018. Molecular Neurodegeneration.)</h4><ul>
<li><strong>Subjects:</strong> <mark>Dorsolateral prefrontal cortex</mark> of symptomatic AD (n = 20), symptomatic AD (AsymAD) (n = 14), and control (n = 13) cases.</li>
<li><strong>Methods:</strong> isobaric tandem mass tag (TMT) mass spectrometry</li>
<li><strong>Results:</strong> 350 proteins had altered levels between AsymAD and AD not caused by changes in specific cell type abundance, potentially reflecting biochemical changes that are associated with cognitive decline in AD. RNA binding proteins were enriched in network modules that correlated with AD pathology. </li>
</ul>
<h4 id="3-Quantitative-proteomics-reveals-distinct-composition-of-amyloid-plaques-in-Alzheimer’s-disease-2018-Alzheimer’s-disease-amp-Dementia"><a href="#3-Quantitative-proteomics-reveals-distinct-composition-of-amyloid-plaques-in-Alzheimer’s-disease-2018-Alzheimer’s-disease-amp-Dementia" class="headerlink" title="3. Quantitative proteomics reveals distinct composition of amyloid plaques in Alzheimer’s disease. (2018. Alzheimer’s disease &amp; Dementia.)"></a>3. <a target="_blank" rel="noopener" href="https://alz-journals.onlinelibrary.wiley.com/doi/10.1016/j.jalz.2018.10.006">Quantitative proteomics reveals distinct composition of amyloid plaques in Alzheimer’s disease</a>. (2018. Alzheimer’s disease &amp; Dementia.)</h4><ul>
<li><strong>Subjects:</strong> <br> Human: <mark>Hippocampal (HC) tissue</mark> from three AD donors and three age-matched non-AD donors was used for plaque capture. <mark>Hippocampal and temporal lobe (TL) tissue</mark> from another 12 donors without neurodegenerative disease was used for human aging brain proteome determination.<br> Mouse: Whole brains from 6 APP/PS1 model mice and 3 WT mice (all 50 weeks old) were dissected for plaque capture. HC and TL from 4 WT mice in each age group (20, 50, 120 weeks) were dissected for mouse aging brain proteome determination.</li>
<li><strong>Methods:</strong> APs and adjacent control regions were collected from fresh-frozen brain sections using laser capture dissection. Tag-labeling coupled high-throughput mass spectra.</li>
<li><strong>Results:</strong> Over 4000 proteins were accurately quantified, and more than 40 were identified as highly enriched in both AD and non-AD APs, including apoE, midkine, VGFR1, and complement C4.</li>
</ul>
<h4 id="4-Proteomic-Atlas-of-the-Human-Brain-in-Alzheimer’s-Disease-2019-Jounral-of-Proteome-Research"><a href="#4-Proteomic-Atlas-of-the-Human-Brain-in-Alzheimer’s-Disease-2019-Jounral-of-Proteome-Research" class="headerlink" title="4. Proteomic Atlas of the Human Brain in Alzheimer’s Disease. (2019. Jounral of Proteome Research.)"></a>4. <a target="_blank" rel="noopener" href="https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00004">Proteomic Atlas of the Human Brain in Alzheimer’s Disease</a>. (2019. Jounral of Proteome Research.)</h4><ul>
<li><strong>Subjects:</strong> Nine brain sections: <mark>Amygdala (AMY), Caudate Nucleus (CNC), Cerebellum (CBM), Entorhinal Cortex (ECX), Inferior Parietal Lobule (IPL), Middle Frontal Gyrus (MFG), Superior Temporal Gyrus (STG), Thalamus (THA), and Visual Cortex (VCX)</mark> in three individuals (The “no tangle” brain (0) contained no observable tangles upon dissection and is not afflicted with Alzheimer’s disease. The “intermediate tangle” brain (+) was categorized as stage III with tangles identified primarily in the entorhinal region. The “severe Tangle” brain (++) falls into the stage VI category with tau aggregates widespread and likely resulting in isocortical destruction.)</li>
<li><strong>Methods:</strong> tandem mass spectrometry</li>
<li><strong>Results:</strong> AD affected tissue exhibited slightly elevated levels of tau protein with similar relative expression to factors associated with the AD pathology. </li>
</ul>
<h4 id="5-Molecular-Network-Analysis-of-the-Urinary-Proteome-of-Alzheimer’s-Disease-Patients-2019-Dementia-amp-Geriatric-Cognitive-Disorders"><a href="#5-Molecular-Network-Analysis-of-the-Urinary-Proteome-of-Alzheimer’s-Disease-Patients-2019-Dementia-amp-Geriatric-Cognitive-Disorders" class="headerlink" title="5. Molecular Network Analysis of the Urinary Proteome of Alzheimer’s Disease Patients. (2019. Dementia &amp; Geriatric Cognitive Disorders.)"></a>5. <a target="_blank" rel="noopener" href="https://www.karger.com/Article/FullText/496100">Molecular Network Analysis of the Urinary Proteome of Alzheimer’s Disease Patients</a>. (2019. Dementia &amp; Geriatric Cognitive Disorders.)</h4><ul>
<li><strong>Subjects:</strong> <mark>Urine samples</mark> collected from 18 AD patients and 18 age- and sex-matched cognitively normal controls.</li>
<li><strong>Methods:</strong> Mass spectrometry and semi-quantified with the normalized spectral index method.</li>
<li><strong>Results:</strong> The levels of 109 proteins significantly differed between AD patients and controls. Among these, annotation clusters related to lysosomes, complement activation, and gluconeogenesis were significantly enriched.</li>
</ul>
<h4 id="6-Comparative-profiling-of-the-synaptic-proteome-from-Alzheimer’s-disease-patients-with-focus-on-the-APOE-genotype-2019-Acta-Neuropathologica-Communications"><a href="#6-Comparative-profiling-of-the-synaptic-proteome-from-Alzheimer’s-disease-patients-with-focus-on-the-APOE-genotype-2019-Acta-Neuropathologica-Communications" class="headerlink" title="6. Comparative profiling of the synaptic proteome from Alzheimer’s disease patients with focus on the APOE genotype. (2019.Acta Neuropathologica Communications.)"></a>6. <a target="_blank" rel="noopener" href="https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-019-0847-7">Comparative profiling of the synaptic proteome from Alzheimer’s disease patients with focus on the APOE genotype</a>. (2019.Acta Neuropathologica Communications.)</h4><ul>
<li><strong>Subjects:</strong> AD, APOE3/3, <mark>primary visual cortex</mark>, n=8; AD, APOE3/4, <mark>primary visual cortex</mark>, n=11; AD, APOE3/3, <mark>superior temporal gyrus</mark>, n=7; AD, APOE3/4, <mark>superior temporal gyrus</mark>, n=9 ; non-demented controls, APOE3/3, primary visual cortex, n=8; non-demented controls, APOE3/4, primary visual cortex, n=5; non-demented controls, APOE3/3, superior temporal gyrus, n=8; non-demented controls, APOE3/4, superior temporal gyrus, n=7.</li>
<li><strong>Methods:</strong> complex 8-plex TMT LC-MS/MS analysis.</li>
<li><strong>Results:</strong> Proteins involved in glutamatergic synaptic signalling and synaptic plasticity are decreased in AD with temporal cortex (which has high levels of pathology) being more severely affected than occipital cortex (which has lower levels of pathology) and APOE4 carriers more affected than APOE3 carriers.</li>
</ul>
<h4 id="7-Rapamycin-regulates-cholesterol-biosynthesis-and-cytoplasmic-ribosomal-proteins-in-hippocampus-and-temporal-lobe-of-APP-PS1-mouse-2019-Journal-of-Neurological-Sciences"><a href="#7-Rapamycin-regulates-cholesterol-biosynthesis-and-cytoplasmic-ribosomal-proteins-in-hippocampus-and-temporal-lobe-of-APP-PS1-mouse-2019-Journal-of-Neurological-Sciences" class="headerlink" title="7. Rapamycin regulates cholesterol biosynthesis and cytoplasmic ribosomal proteins in hippocampus and temporal lobe of APP/PS1 mouse. (2019. Journal of Neurological Sciences.)"></a>7. <a target="_blank" rel="noopener" href="https://www.jns-journal.com/article/S0022-510X(19)30083-8/fulltext">Rapamycin regulates cholesterol biosynthesis and cytoplasmic ribosomal proteins in hippocampus and temporal lobe of APP/PS1 mouse</a>. (2019. Journal of Neurological Sciences.)</h4><ul>
<li><strong>Subjects:</strong> proteome profles of hippocampus and temporal lobe of wild-type mice, APP/PS1 mice and rapamycintreated APP/PS1 mice.</li>
<li><strong>Methods:</strong> HPLC-coupled high-throughput mass spectrometry approach.</li>
<li><strong>Results:</strong> Proteome analysis identified 100 significantly changed (SC) proteins in hippocampus and 260 in temporal lobe in APP/PS1 mice. Among these, 57 proteins in hippocampus and 167 proteins in temporal lobe were rescued by rapamycin.</li>
</ul>
<h4 id="8-Large-scale-proteomic-analysis-of-Alzheimer’s-disease-brain-and-cerebrospinal-fluid-reveals-early-changes-in-energy-metabolism-associated-with-microglia-and-astrocyte-activation-2020-Nature-Medicine"><a href="#8-Large-scale-proteomic-analysis-of-Alzheimer’s-disease-brain-and-cerebrospinal-fluid-reveals-early-changes-in-energy-metabolism-associated-with-microglia-and-astrocyte-activation-2020-Nature-Medicine" class="headerlink" title="8. Large-scale proteomic analysis of Alzheimer’s disease brain and cerebrospinal fluid reveals early changes in energy metabolism associated with microglia and astrocyte activation. (2020. Nature Medicine.)"></a>8. <a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41591-020-0815-6">Large-scale proteomic analysis of Alzheimer’s disease brain and cerebrospinal fluid reveals early changes in energy metabolism associated with microglia and astrocyte activation</a>. (2020. Nature Medicine.)</h4><ul>
<li><strong>Subjects and methods:</strong> (1) <mark>dorsolateral prefrontal cortex (DLPFC)</mark> tissue in 44 cases from the Baltimore Longitudinal Study of Aging (BLSA), 178 cases from the Banner Sun Health Research Institute (Banner), 166 cases from the Mount Sinai School of Medicine Brain Bank (MSSB) and 65 cases from the Adult Changes in Thought Study (ACT), for a total of 453 control, asymptomatic AD (AsymAD) and AD brains;  MS-based proteomics using label-free quantitation (LFQ). (2) 340 <mark>DLPFC</mark> brain tissues from a community-based aging cohort, the Religious Orders Study and Memory and Aging Project (ROS/MAP); with a different MS-based protein quantification approach using isobaric multiplex tandem mass tags (TMTs). (3) control and AD brain tissue from the <mark>temporal cortex</mark> in a separate set of 111 brains from the Mayo Clinic and control, AsymAD and AD brain tissue from precuneus in the same set of brains from the BLSA; LFQ-MS. (4) <mark>DLPFC</mark> tissues from Johns Hopkins in 84 individuals aged 30 to 69 years; LFQ-MS. (5) 331 <mark>DLPFC</mark> tissues from control individuals and those with AD, amyotrophic lateral sclerosis (ALS), frontotemporal lobar degeneration with TDP-43 pathology (FTLD-TDP), progressive supranuclear palsy (PSP), corticobasal degeneration (CBD), Parkinson’s disease and Parkinson’s disease dementia (PD/PDD) and multiple systems atrophy (MSA); LFQ-MS. (6) <mark>cerebrospinal fluid</mark> (CSF) from two separate cohorts: one cohort of 297 individuals consisting of controls and patients with AD (Cohort 1) and a second cohort of 96 individuals classified into control, AsymAD and AD (Cohort 2); TMT-MS.</li>
<li><strong>Results:</strong> A protein network module linked to sugar metabolism emerged as one of the modules most significantly associated with AD pathology and cognitive impairment. Proteins from this module were elevated in cerebrospinal fluid in early stages of the disease. </li>
</ul>
<h4 id="9-Integrated-proteomics-reveals-brain-based-cerebrospinal-fluid-biomarkers-in-asymptomatic-and-symptomatic-Alzheimer’s-disease-2020-Science-Advance"><a href="#9-Integrated-proteomics-reveals-brain-based-cerebrospinal-fluid-biomarkers-in-asymptomatic-and-symptomatic-Alzheimer’s-disease-2020-Science-Advance" class="headerlink" title="9. Integrated proteomics reveals brain-based cerebrospinal fluid biomarkers in asymptomatic and symptomatic Alzheimer’s disease. (2020. Science Advance.)"></a>9. <a target="_blank" rel="noopener" href="https://n.neurology.org/content/94/15_Supplement/4738">Integrated proteomics reveals brain-based cerebrospinal fluid biomarkers in asymptomatic and symptomatic Alzheimer’s disease</a>. (2020. Science Advance.)</h4><ul>
<li><strong>Subjects:</strong> <mark>cerebrospinal fluid</mark> (CSF) : discovery cohort: 20 cognitively normal individuals and 20 individuals with AD; CSF replication 1 included samples from 32 healthy controls, 31 individuals with AsymAD, and 33 individuals with AD. CSF replication 2 contained 147 control and 150 AD samples. </li>
<li>Brain tissue: <mark>DLPFC</mark> 10 healthy control, 10 PD, 10 AD/PD, and 10 AD cases. The replication brain cohort included 19 cases identical to the discovery cohort (10 control and 9 AD cases), as well as 8 AsymAD cases unique to this cohort.</li>
<li><strong>Methods:</strong> multiplex tandem mass tag (TMT) labeling mass spectrometry.</li>
<li><strong>Results:</strong> Multiplex mass spectrometry identified ~3500 and ~12,000 proteins in AD CSF and brain, respectively. Synaptic and metabolic panels were decreased in AD brain but increased in CSF, while glial-enriched myelination and immunity panels were increased in brain and CSF.</li>
</ul>
<h4 id="10-Deep-Multilayer-Brain-Proteomics-Identifies-Molecular-Networks-in-Alzheimer’s-Disease-Progression-2020-Neuron"><a href="#10-Deep-Multilayer-Brain-Proteomics-Identifies-Molecular-Networks-in-Alzheimer’s-Disease-Progression-2020-Neuron" class="headerlink" title="10. Deep Multilayer Brain Proteomics Identifies Molecular Networks in Alzheimer’s Disease Progression. (2020. Neuron.)"></a>10. <a target="_blank" rel="noopener" href="https://alz-journals.onlinelibrary.wiley.com/doi/abs/10.1002/alz.037231">Deep Multilayer Brain Proteomics Identifies Molecular Networks in Alzheimer’s Disease Progression</a>. (2020. Neuron.)</h4><ul>
<li><strong>Subjects:</strong> <mark>Frontal cortical</mark> samples of 100 human cases and selected 90 samples of high quality for profiling the whole proteome and phosphoproteome in 5 equal groups: (1) controls with a low pathology of plaques and tangles (LPCs), (2) controls with high Ab pathology but no detectable cognitive defects (HPCs), (3) mild cognitive impairment (MCI) with Ab pathology and a slight but measurable defect in cognition, (4) late-stage AD with high pathology scores of plaques and tangles, and (5) progressive supranuclear palsy (PSP), another neurodegenerative disorder of tauopathy.</li>
<li><strong>Methods:</strong> Multiplexed TMT method and two-dimensional LC/LC-MS/MS.</li>
<li><strong>Results:</strong> Highlighting 173 protein changes in 17 pathways. Determine 15 Ab-correlated proteins (e.g., MDK, NTN1, SMOC1, SLIT2, and HTRA1). Multi-omics integration prioritizes AD-related molecules and pathways, including amyloid cascade, inflammation, complement, WNT signaling, TGF-b and BMP signaling, lipid metabolism, iron homeostasis, and membrane transport.</li>
</ul>
<h4 id="11-Proteomic-and-Transcriptomic-Analyses-Reveal-Pathological-Changes-in-the-Entorhinal-Cortex-Region-that-Correlate-Well-with-Dysregulation-of-Ion-Transport-in-Patients-with-Alzheimer’s-Disease-2021-Molecular-Neurobiology"><a href="#11-Proteomic-and-Transcriptomic-Analyses-Reveal-Pathological-Changes-in-the-Entorhinal-Cortex-Region-that-Correlate-Well-with-Dysregulation-of-Ion-Transport-in-Patients-with-Alzheimer’s-Disease-2021-Molecular-Neurobiology" class="headerlink" title="11. Proteomic and Transcriptomic Analyses Reveal Pathological Changes in the Entorhinal Cortex Region that Correlate Well with Dysregulation of Ion Transport in Patients with Alzheimer’s Disease. (2021. Molecular Neurobiology.)"></a>11. <a target="_blank" rel="noopener" href="https://link.springer.com/article/10.1007/s12035-021-02356-3">Proteomic and Transcriptomic Analyses Reveal Pathological Changes in the Entorhinal Cortex Region that Correlate Well with Dysregulation of Ion Transport in Patients with Alzheimer’s Disease.</a> (2021. Molecular Neurobiology.)</h4><ul>
<li><strong>Subjects:</strong> AD <mark>entorhinal cortex</mark> (EC) samples and non-AD control EC samples. (RNA-seq: 7 AD, 18 control; proteome: 4 AD, 4 control.)</li>
<li><strong>Methods:</strong> Transcriptome based on the Illumina RNA-seq and proteome based on TMT-labelling.</li>
<li><strong>Results:</strong> 57 genes were detected both in transcriptome and proteome data, including 51 in similar altering trends (7 upregulated, 44 downregulated) and 6 in inverse trends when compared AD vs. control.</li>
</ul>
<h4 id="12-Quantitative-proteomic-analysis-of-the-frontal-cortex-in-Alzheimer’s-disease-2020-Neurochemistry"><a href="#12-Quantitative-proteomic-analysis-of-the-frontal-cortex-in-Alzheimer’s-disease-2020-Neurochemistry" class="headerlink" title="12. Quantitative proteomic analysis of the frontal cortex in Alzheimer’s disease. (2020. Neurochemistry.)"></a>12. <a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1111/jnc.15116">Quantitative proteomic analysis of the frontal cortex in Alzheimer’s disease</a>. (2020. Neurochemistry.)</h4><ul>
<li><strong>Subjects:</strong> <mark>frontal cortex</mark> from AD patients with corresponding age-matched brain samples.</li>
<li><strong>Methods:</strong> Tandem mass tags (TMT) approach coupled to highresolution mass spectrometry.</li>
<li><strong>Results:</strong> LC-MS/MS analysis carried out on an Orbitrap Fusion Lumos Tribrid mass spectrometer led to identification of 8,066 proteins. Of these, 432 proteins were observed to be significantly altered (&gt;1.5 fold) in their expression in AD brains.</li>
</ul>
<h3 id="References"><a href="#References" class="headerlink" title="References"></a><strong>References</strong></h3><ul>
<li><a target="_blank" rel="noopener" href="https://pubs.acs.org/doi/10.1021/pr401202d">https://pubs.acs.org/doi/10.1021/pr401202d</a></li>
<li><a target="_blank" rel="noopener" href="https://molecularneurodegeneration.biomedcentral.com/articles/10.1186/s13024-018-0282-4">https://molecularneurodegeneration.biomedcentral.com/articles/10.1186/s13024-018-0282-4</a></li>
<li><a target="_blank" rel="noopener" href="https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00004">https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00004</a></li>
<li><a target="_blank" rel="noopener" href="https://www.karger.com/Article/FullText/496100">https://www.karger.com/Article/FullText/496100</a></li>
<li><a target="_blank" rel="noopener" href="https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-019-0847-7">https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-019-0847-7</a></li>
<li><a target="_blank" rel="noopener" href="https://www.jns-journal.com/article/S0022-510X(19)30083-8/fulltext">https://www.jns-journal.com/article/S0022-510X(19)30083-8/fulltext</a></li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41591-020-0815-6">https://www.nature.com/articles/s41591-020-0815-6</a></li>
<li><a target="_blank" rel="noopener" href="https://n.neurology.org/content/94/15_Supplement/4738">https://n.neurology.org/content/94/15_Supplement/4738</a></li>
<li><a target="_blank" rel="noopener" href="https://alz-journals.onlinelibrary.wiley.com/doi/abs/10.1002/alz.037231">https://alz-journals.onlinelibrary.wiley.com/doi/abs/10.1002/alz.037231</a></li>
<li><a target="_blank" rel="noopener" href="https://link.springer.com/article/10.1007/s12035-021-02356-3">https://link.springer.com/article/10.1007/s12035-021-02356-3</a></li>
<li><a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1111/jnc.15116">https://onlinelibrary.wiley.com/doi/10.1111/jnc.15116</a></li>
</ul>

    </div>

    
    
    

    <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/Alzheimer-s-disease/" rel="tag"># Alzheimer's disease</a>
              <a href="/tags/proteome-sequencing/" rel="tag"># proteome sequencing</a>
          </div>

        

          <div class="post-nav">
            <div class="post-nav-item">
            </div>
            <div class="post-nav-item">
                <a href="/2022/01/25/IGV-genome-browser-usage/" rel="next" title="IGV genome browser usage">
                  IGV genome browser usage <i class="fa fa-chevron-right"></i>
                </a>
            </div>
          </div>
    </footer>
  </article>
</div>






</div>
  </main>

  <footer class="footer">
    <div class="footer-inner">


<div class="copyright">
  &copy; 
  <span itemprop="copyrightYear">2022</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Lin</span>
</div>
  <div class="powered-by">Powered by <a href="https://hexo.io/" rel="noopener" target="_blank">Hexo</a> & <a href="https://theme-next.js.org/" rel="noopener" target="_blank">NexT.Gemini</a>
  </div>

    </div>
  </footer>

  
  <script src="https://cdn.jsdelivr.net/npm/animejs@3.2.1/lib/anime.min.js" integrity="sha256-XL2inqUJaslATFnHdJOi9GfQ60on8Wx1C2H8DYiN1xY=" crossorigin="anonymous"></script>
<script src="/js/comments.js"></script><script src="/js/utils.js"></script><script src="/js/motion.js"></script><script src="/js/next-boot.js"></script>

  





  





</body>
</html>
